x/txfees/types/constants.go

```diff
@@ -0,0 +1,7 @@
+package types
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+// ConsensusMinFee is a governance set parameter from prop 354 (https://www.mintscan.io/osmosis/proposals/354)
+// Its intended to be .0025 uosmo / gas
+var ConsensusMinFee sdk.Dec = sdk.NewDecWithPrec(25, 4)
```

Comment:
I was thinking of an approach using params so that the governance can revisit this if necessary in the future as well, I do think that constant should be good for now

Reply:
I don't want to bother with params for now, since I would like to just get to EIP-1559 within 3 months